open_spiel/games/pig.cc

```diff
@@ -201,7 +201,7 @@ void PigState::DoApplyAction(Action move) {
   } else if (IsChanceNode()) {
     // Resolve chance node outcome. If 1, reset turn total and change players;
     // else, add to total and keep going.
-    if (move == 1) {
+    if (move == 0) {
```

Comment:
It seems that this is the only needed change. The following line already increments the `move`.

https://github.com/deepmind/open_spiel/blob/8c8e0863380acaf93fe0b00780196d9117eaac66/open_spiel/games/pig.cc#L211

Reply:
Yep, this looks good.